pandas/tests/util/test_assert_series_equal.py

```diff
@@ -115,7 +117,10 @@ def test_less_precise(data1, data2, dtype, decimals):
     if (decimals == 5 or decimals == 10) or (
         decimals >= 3 and abs(data1 - data2) >= 0.0005
     ):
-        msg = "Series values are different"
+        if is_extension_array_dtype(dtype):
```

Comment:
why would this message change? the Series are still different

Reply:
```
In [6]: pd.testing.assert_series_equal(pd.Series([1]), pd.Series([2]))
...

AssertionError: Series are different

Series values are different (100.0 %)
[index]: [0]
[left]:  [1]
[right]: [2]
```

```
In [7]: pd.testing.assert_series_equal(pd.Series([1], dtype="Int64"), pd.Series(
   ...: [2], dtype="Int64"))
...

AssertionError: ExtensionArray are different

ExtensionArray values are different (100.0 %)
[index]: [0]
[left]:  [1]
[right]: [2]
```